ddtrace/internal/telemetry/data/request_body.py

```diff
@@ -0,0 +1,23 @@
+from datetime import datetime
+import json
+from typing import Optional
+
+from ...runtime import get_runtime_id
+from .application import Application
+from .host import Host
+from .payload import Payload
+
+
+class RequestBody:
+    def __init__(self, payload, application=None, host=None):
+        self.tracer_time = int(datetime.now().timestamp())  # type: int
+        self.runtime_id = get_runtime_id()  # type: str
+        self.api_version = "v1"  # type: str
+        self.seq_id = 0  # type: int
+        self.application = application or Application()  # type: Application
+        self.host = host or Host()  # type: Host
+        self.payload = payload  # type: Optional[Payload]
+        self.request_type = payload.REQUEST_TYPE  # type: str
+
+    def to_json(self):
+        return json.dumps(self, default=lambda o: o.__dict__)
```

Comment:
I would avoid doing `o.__dict__` here, this is a very heavy way of just manually building/defining the dict to be created.